.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        gemfile:
+          - Gemfile
+          - Gemfile.5.2
+          - Gemfile.6.0
+          - Gemfile.6.1
```

Comment:
Should we add a 7.0 and 7.1?

Reply:
The current Gemfile is for the top version of 7.x.x, I did not see much difference between 7.0 and 7.1 for our use case. So, I am not planning to add them separately. But let me know if you find otherwise.